### DISCUSSION TITLE
BS Swiper fatal error -- undefined function bootscore_category_badge()

### DISCUSSION THREAD

**Discussion by antforager** [bootscore/bootscore#509] [Q&A]
I've been getting this for a week or so in my logs but my site still works so it hasn't been critical. I have tried it with 5.1 and 5.2.  Here is a pretty screenshot of the log and the stack trace is cut and pasted below.

<img width="434" alt="image" src="https://github.com/bootscore/bootscore/assets/11689112/db2be6d2-5537-4263-a842-f6515c96c1cd">

Fatal error: Uncaught Error: Call to undefined function bootscore_category_badge() in /var/www/wp-content/plugins/bs-swiper-main/templates/sc-swiper-card.php:102

Stack Trace
1.	
{anonymous}()
/var/www/wp-content/plugins/bs-swiper-main/templates/sc-swiper-card.php:102
2.	
bootscore_swiper(Array(6), '', 'bs-swiper-card')
/var/www/wp-includes/shortcodes.php:355
3.	
do_shortcode_tag(Array(7))
4.	
preg_replace_callback('/\[(\[?)(contact\-form...', 'do_shortcode_tag', ' <section class="bg-se...')
/var/www/wp-includes/shortcodes.php:227
5.	
do_shortcode(' <section class="bg-se...')
/var/www/wp-includes/class-wp-hook.php:308
6.	
WP_Hook->apply_filters(' <section class="bg-se...', Array(1))
/var/www/wp-includes/plugin.php:205
7.	
apply_filters('the_content', ' <sect...')
/var/www/wp-includes/post-template.php:255
8.	
the_content()
/var/www/wp-content/themes/byvex-woocommerce-starter/template-parts/content-page.php:4
9.	
require('/var/www/wp-content/th...')
/var/www/wp-includes/template.php:785
10.	
load_template('/var/www/wp-content/th...', false, Array(0))
/var/www/wp-includes/template.php:718
11.	
locate_template(Array(1), true, false, Array(0))
/var/www/wp-includes/general-template.php:204
12.	
get_template_part('template-parts/content...')
/var/www/wp-content/themes/byvex-woocommerce-starter/page.php:7
13.	
include('/var/www/wp-content/th...')
/var/www/wp-includes/template-loader.php:106
14.	
require_once('/var/www/wp-includes/t...')
/var/www/wp-blog-header.php:19
15.	
require('/var/www/wp-blog-heade...')
/var/www/index.php:17
Context
URL	/
HTTP Method	GET
HTTP Host	antgear.com
HTTP Status	200
isSSL	Yes
Current Filter	the_content
Execution Time	0.817 s
Memory Usage	32.07 MiB
PHP SAPI	cgi-fcgi

**Comment by crftwrk** [comment]
You can ignore it. `<?php bootscore_category_badge(); ?>` shows the post categories. In your case, it seems that you show products in post card template. Products have no category, they use a taxonomy. So, you can easily override template in your child and remove this line. But why not use the `bs-swiper-card-product` shortcode instead?

See https://bootscore.me/documentation/bs-swiper/
